imports/startup/server/index.coffee

```diff
@@ -145,6 +145,26 @@ Meteor.methods
       """
     makeRequest(query)
 
+  'getTrendingInfectiousAgents': (date) ->
+    query = prefixes + """
+      SELECT ?resolvedTerm
```

Comment:
This isn't quite what I had in mind for computing the trending agent. Let's define the trending score for agent with M mentions over the last D days and M2 mentions over the last D2 days as (M/D)/(M2/D2). D is the trending interval and D2 is some multiple of D.